go/inst/instance.go

```diff
@@ -255,12 +257,17 @@ func (this *Instance) IsMaster() bool {
 
 // ReplicaRunning returns true when this instance's status is of a replicating replica.
 func (this *Instance) ReplicaRunning() bool {
-	return this.IsReplica() && this.Slave_SQL_Running && this.Slave_IO_Running
+	return this.IsReplica() && this.ReplicationSQLThreadState.IsRunning() && this.ReplicationIOThreadState.IsRunning()
 }
 
 // NoReplicationThreadRunning returns true when neither SQL nor IO threads are running (including the case where isn't even a replica)
```

Comment:
How does this return true if the host isn't a replica? `IsStopped()` maps to `ReplicationThreadStateStopped` but doesn't include `ReplicationThreadStateNoThread`? Or am I reading this incorrectly?